example/spring-data-mongodb/src/main/kotlin/com/github/inflab/example/spring/data/mongodb/repository/WildcardSearchRepository.kt

```diff
@@ -0,0 +1,55 @@
+package com.github.inflab.example.spring.data.mongodb.repository
+
+import com.github.inflab.example.spring.data.mongodb.entity.mflix.Movies
+import com.github.inflab.spring.data.mongodb.core.aggregation.aggregation
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.aggregate
+import org.springframework.data.mongodb.core.aggregation.AggregationResults
+import org.springframework.stereotype.Repository
+
+@Repository
+class WildcardSearchRepository(
+    private val mongoTemplate: MongoTemplate,
+) {
+    data class TitleDto(val title: String)
+
+    /**
+     * @see <a href="https://www.mongodb.com/docs/atlas/atlas-search/wildcard/#examples">Example</a>
+     */
+    fun findTitleWithGreenD(): AggregationResults<TitleDto> {
+        val aggregation = aggregation {
+            search {
+                wildcard {
+                    path(Movies::title)
+                    query("Green D*")
+                }
+            }
+            project {
```

Comment:
please add this this between stage.
I will add `$limit` stage after it is implemented. 

```kt

// TODO: add $limit stage

```

Reply:
I have found that this example does not have `$limit` 
my mistake :)